### ISSUE TITLE
[BUG] Missing Space Between `@DefaultEmpty` Annotation and `var` Keyword in Macro Expansion

### ISSUE THREAD

**Issue by ethan-wickstrom** [DataDog/swift-jsonapi#10]
A bug in the `@ResourceWrapper` macro is causing a missing space between the `@DefaultEmpty` annotation and the `var` keyword during macro expansion, resulting in the following errors:

- `Expected 'var' keyword in property declaration`
- `Unknown attribute 'DefaultEmptyvar'`

This issue occurs when trying to use the macro in a struct definition for JSON:API resources. The missing space causes the attribute to be incorrectly recognized, leading to a compilation failure.

**Checklist**
- [X] I can reproduce this issue with a vanilla Swift project.
- [X] I can reproduce this issue using the `main` branch of this package.
- [X] This bug hasn't been addressed in an [existing GitHub issue](https://github.com/Datadog/swift-jsonapi/issues).

**Steps to reproduce**
1. Create a Swift project and include the `swift-jsonapi` package.
2. Define a struct using the `@ResourceWrapper` macro and include a property with the `@ResourceAttribute` annotation.
3. Compile the project to observe the missing space between the `@DefaultEmpty` annotation and the `var` keyword, leading to the error.
   
   Here’s an example:

   ```swift
   @ResourceWrapper(type: "roles")
   struct Role: Equatable {
       var id: String
       
       @ResourceAttribute var name: String
       @ResourceAttribute var permissions: [String]
       @ResourceRelationship var users: [User]
       @ResourceAttribute var createdAt: Date
       @ResourceAttribute var updatedAt: Date
       @ResourceAttribute var deletedAt: Date?
       @ResourceAttribute var createdBy: String?
       @ResourceAttribute var updatedBy: String?
       @ResourceAttribute var deletedBy: String?
   }
   ```

   After macro expansion:

   ```swift
   struct Attributes: Equatable, Codable {
       var name: String
       @DefaultEmptyvar permissions:[String]
       var createdAt: Date
       var updatedAt: Date
       var deletedAt: Date?
       var createdBy: String?
       var updatedBy: String?
       var deletedBy: String?
   }
   ```

**Expected behavior**
The macro expansion should correctly place a space between `@DefaultEmpty` and `var`, avoiding any compilation errors. The correct expansion should be:

```swift
@DefaultEmpty var permissions: [String]
```

**Version information**
- JSONAPI: 0.1.0
- OS: macOS 15.0
- Xcode: 16.0 beta 4

**Additional context**
The issue seems to be a simple string formatting problem during macro expansion. Fixing this would ensure the attributes are correctly recognized and reduce potential errors in projects using the package. Also, when `var` is swapped with `let`, the same issue happens, except it will say `@DefaultEmptylet permissions: [String]`

**Comment by ethan-wickstrom** [comment]
I'm following up on this issue to see if there's been any progress, if there's any additional information I could provide to help resolve it, or if I can get my hands dirty and help out with anything.

This fix would be incredibly helpful for ongoing work with the package, so any update would be greatly appreciated.

**Comment by gonzalezreal** [comment]
Hi @ethan-wickstrom,
Can you reproduce this issue with Xcode 15.4 or it happens only in Xcode 16?

**Comment by gonzalezreal** [comment]
Fixed in #11. In the meantime, you can add a new line after `@ResourceAttribute` to workaround the issue.

```swift
@ResourceAttribute
var permissions: [String]
```
